reliability/config/example_reliability.yaml

```diff
@@ -29,6 +29,38 @@ reliability:
     # you must be a member of the slack channel to receive the notification.
     slack_member: <Your slack member id>
 
+    krakenIntegration:
```

Comment:
It looks like this section is tabbed over too much. I think this title should be even with the slackIntegration section above right?

Reply:
@paigerube14 You caught it.